src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1196,7 +1186,7 @@ ExpressionPtr Visitor::visitTypesafe(Parser::AggregateContext* ctx) {
 
     std::string separator;
     if (ctx->string()) {
-      separator = ctx->string()->getText();
+      separator = visitTypesafe(ctx->string());
```

Comment:
After doing some reading, 
I found that this is broken in a subtle way:
The string rule also allows `"""tripleQuotes"""` with different escaping rules,
which are currently not handled in the `visitString` rule. This should be parsed in to a typesafe format that has a unique representation. I have something like this for the TurtleParser (the `rdfEscaping` module, but I am not sure, where this is used currently in the SparqlParser.
Maybe we can discuss this sometime soon, make a TODO here and at the `parseString` method.